arduino/sketch/profiles.go

```diff
@@ -260,5 +270,7 @@ func LoadProjectFile(file *paths.Path) (*Project, error) {
 	if err := yaml.Unmarshal(data, &res); err != nil {
 		return nil, err
 	}
+	res.Profiles = res.getProfiles()
```

Comment:
Can't we do this inside the `YamlUnmarshal`? We may forget to produce the profiles for example we are trying to do another `yaml.Unmarshal` here: https://github.com/arduino/arduino-cli/blob/f561da0de04593e3eb24acace22eacd77cb084c4/arduino/sketch/yaml.go#L77